include/p4mlir/Transforms/Passes.td

```diff
@@ -141,4 +141,19 @@ def RemoveAliases : Pass<"p4hir-remove-aliases", "mlir::ModuleOp"> {
   let dependentDialects = ["P4MLIR::P4HIR::P4HIRDialect"];
 }
 
+//===----------------------------------------------------------------------===//
+// TupleToStruct
+//===----------------------------------------------------------------------===//
+
+def TupleToStruct : Pass<"p4hir-tuple-to-struct", "mlir::ModuleOp"> {
+  let summary = "Convert p4hir.tuple types/ops into p4hir.stract types/ops";
+  let description = [{
+    Eliminate all p4hir.tuple types and operations by converting them into equivalent
```

Comment:
Well, there are no `p4hir.tuple` types actually... Also, will you please correct the typos?